episodes/access_hub.md

```diff
@@ -0,0 +1,204 @@
+(episode-access)=
+# Controlling access to your hub
+
+teaching: 15
+exercises: 15
+
+:::::{card} 
+OVERVIEW
+^^^
+
+::::{grid}
+:gutter: 2
+
+:::{grid-item}
+Questions
+
+- Who has control over access to a hub?
+- How do I invite members to a GitHub Organization?
+- How do I grant access to a hub with GitHub Teams?
+
+:::
+
+:::{grid-item}
+Objectives
+
+- Control user access to a hub via GitHub Organizations and Teams
+- Add users to a GitHub Organization
+- Add and remove users from a GitHub Team to grant access to a hub
+
+:::
+
+:::::
+
+## Who decides which users can access a hub?
+
+Hub Champions can control access to the hub by adding users to their {ref}`GitHub Organization <glossary:github-org>` and {ref}`Team<glossary:github-team>`. Each hub user needs to [sign up](https://github.com/signup) for an individual account on GitHub.
+
+A Hub Champion should have Owner permissions for their GitHub Organization in order to invite non-members to it. Inviting a user to the organization does not automatically grant the user access to a hub. A member of a GitHub Organization must be added to a GitHub Team associated with the hub in order to log into the hub with their GitHub credentials.
+
+:::{note}
+Your GitHub Team is defined and specially linked to your hub when the hub was initally deployed. This information should be known to Hub Champions. If not, please contact your community representative who deployed the hub.
+:::
+
+Follow the guidance below to practise granting user access to the [Community Showcase Hub](https://showcase.2i2c.cloud/), specifically
+
+- adding (but *not* removing for now) users to a GitHub Organization
+- adding and removing users to and from a GitHub Team.
+
+:::{danger}
+Please refrain from removing other users from the GitHub Organization during the training! The steps in {ref}`Removing members from the organization <org-remove>` are for completeness when it comes to administering your own hub outside of the training.
+
+:::
+
+## Managing your GitHub Organization
+
+- View a list of all of the GitHub Organizations that you are a member of by visiting [https://github.com/settings/organizations](https://github.com/settings/organizations).
+
+:::{figure} ../media/episodes/administer_hub/gh-org-1.png
+:width: 100%
+:alt: Screenshot showing a list of GitHub Organizations on https://github.com/settings/organizations.
+
+Screenshot showing an example list of GitHub Organizations on [https://github.com/settings/organizations](https://github.com/settings/organizations).
+:::
+
+- Click on the link to the [2i2c-community-showcase](https://github.com/2i2c-community-showcase) GitHub Organization (if you cannot see this link then please contact your instructor for access).
```

Comment:
What do you mean by "cannot see this link" here? If it is about authentication/access, perhaps 

```suggestion
- Click on the link to the [2i2c-community-showcase](https://github.com/2i2c-community-showcase) GitHub Organization (if you cannot access the page at this link then please contact your instructor for access).
```

would be clearer?

Reply:
Oops yes thank you for correcting my phrasing!